Spanish/02_Day_Variables_builtin_functions/02_variables_builtin_functions.md

```diff
@@ -0,0 +1,300 @@
+<div align="center">
+  <h1> 30 Dias de Python: Dia 2 - Variables, Funciones integradas</h1>
+  <a class="header-badge" target="_blank" href="https://www.linkedin.com/in/asabeneh/">
+  <img src="https://img.shields.io/badge/style--5eba00.svg?label=LinkedIn&logo=linkedin&style=social">
+  </a>
+  <a class="header-badge" target="_blank" href="https://twitter.com/Asabeneh">
+  <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/asabeneh?style=social">
+  </a>
+
+<sub>Author:
+<a href="https://www.linkedin.com/in/asabeneh/" target="_blank">Asabeneh Yetayeh</a><br>
+<small> Second Edition: July, 2021</small>
+</sub>
+
+</div>
+
+[<< Dia 1](../readme.md) | [Dia 3 >>](../03_Day_Operators/03_operators.md)
+
+![30DaysOfPython](../../images/30DaysOfPython_banner3@2x.png)
+
+- [📘 Día 2](#-día-2)
+   - [Funciones integradas](#funciones-integradas)
+   - [Variables](#variables)
+     - [Declaración de múltiples variables en una línea](#declaración-de-múltiples-variables-en-una-línea)
+   - [Tipos de datos](#tipos-de-datos)
+   - [Comprobación de tipos de datos y conversión](#comprobación-de-tipos-de-datos-y-conversión)
+   - [Números](#números)
+   - [💻 Ejercicios - Día 2](#💻-ejercicios---día-2)
+     - [Ejercicios: Nivel 1](#ejercicios-nivel-1)
+     - [Ejercicios: Nivel 2](#ejercicios-nivel-2)
+
+# 📘 Dia 2
+
+## Funciones integradas
+
+En Python tenemos muchas funciones integradas. Las funciones integradas están disponibles globalmente para su uso, lo que significa que puede hacer uso de las funciones integradas sin importarlas ni configurarlas. Algunas de las funciones integradas de Python más utilizadas son las siguientes: _print()_, _len()_, _type()_, _int()_, _float()_, _str()_, _input()_, _list()_, _dict()_, _min()_, _max()_, _sum()_, _sorted()_, _open()_, _file()_, _help()_ y _dir()_ . En la siguiente tabla, verá una lista exhaustiva de las funciones integradas de Python sacadas de la [documentación de Python](https://docs.python.org/3.9/library/functions.html).
```

Comment:
I think backticks look cleaner than italics for formatting functions. I would link the Spanish version of the docs.